app/api/business/brief/brief_business.py

```diff
@@ -13,6 +16,25 @@ def is_open_to_all(brief):
     return False
 
 
+def can_submit_response_to_brief(brief, user, check_response_limit=True):
+    user_status = BriefUserStatus(brief, user)
+
+    if user_status.has_supplier_errors() or not user_status.is_approved_seller():
+        return False, 'Supplier is invalid'
+
+    if brief.lot.slug != 'atm' or brief.data.get('openTo', '') == 'category':
```

Comment:
Should we use the `is_open_to_all()` helper (brief_business) here?

Reply:
I don't think so, because that considers open to category ATM's to be open to all, which it is in our UI and the daily opp email, but for responding we need to check that the seller is approved in the category for open to category ATMs.